content/en/docs/cluster-administration/platform-settings/notification-management/configure-email.md

```diff
@@ -1,10 +1,62 @@
 ---
-title: "Configure Email"
+title: "Configure Email Notifications"
 keywords: 'KubeSphere, Kubernetes, custom, platform'
-description: ''
-linkTitle: "Configure Email"
+description: 'Configure your email server and add recipients to receive email notifications for alerting policies.'
```

Comment:
to receive email notifications for alerting policies. 
=>
to receive email notifications from alerting policies, kube-events, kube-auditing (or other components in the future).

Reply:
Changed.